src/components/ContributorThumbnail.js

```diff
@@ -189,19 +195,38 @@ const Thumbnail = ({
   } else if (organization.depth === 2) {
     thumbnailImageStyle = classes.thumbnailImage;
   }
+
+  const showtotalChildCount = () => {
+
+    let displayedCount = '';
+
+    if (dropdownLength)
+    {
+      if (filtersActive)
+      {
+        displayedCount = `(${dropdownLength} / ${organization.totalCount})`;
+      }
+      else
+      {
+        displayedCount = `(${dropdownLength})`;
+      }
+    }
+    return displayedCount;
+  };
+
```

Comment:
I also recommend rewriting this to be smaller, but that is up to you.

```suggestion

  const showtotalChildCount = () => {
    if (dropdownLength && filtersActive) {
        return `(${dropdownLength} / ${organization.totalCount})`;
    } else if (dropdownLength) {
        return `(${dropdownLength})`;
    }
  };

```